CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [Unreleased]
 ### Added
+- Lumber - Add databases turnkeys sources.
```

Comment:
It's look pretty cool. But I don't think it should be in that repository 🤔 ? It might be a new repo ? A repo where we can put all our initialisation script and these databases ^^

Reply:
yes, I think about this one: https://github.com/ForestAdmin/forest-examples